.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
```

Comment:
why are you not adding this to the `.git/hooks` directory. Going of the reference at https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks